tools/sigh.ts

```diff
@@ -415,8 +415,11 @@ function makeLink(src: string, dest: string): boolean {
     }
     fs.linkSync(src, dest);
   } catch (lerr) {
-    console.error(`Error linking ${src} to ${dest} ${lerr.message}`);
-    return false;
+    // In case of racing builds.
+    if (!lerr.message.startsWith('EEXIST:')) {
```

Comment:
Thanks for this, two thoughts:
1. Can you file a GH issue to track this so we don't forget?
2. We should probably still log the error even if it's 'EEXIST', but perhaps with a different log level (`warn`?).

Reply:
Sure thing.
I'll file an issue when I get up (it's 2am at my time).

Reply:
done :-)